build/PackageDiffIgnore.xml

```diff
@@ -638,6 +638,9 @@
 			<!-- BEGIN OnWindowCreated -->
 			<Member fullName="Microsoft.UI.Xaml.WindowCreatedEventArgs" reason="Does not exist in WinAppSDK" />
 			<!-- END OnWindowCreated -->
+
+			<Member fullName="Uno.UI.Runtime.Skia.GTK.UI.Text.FontWeightExtensions" reason="Renamed namespace to Gtk instead of GTK" />
+			<Member fullName="Uno.UI.Runtime.Skia.GTK.Extensions.Helpers.CssProviderExtensions" reason="Renamed namespace to Gtk instead of GTK" />
```

Comment:
@MartinZikmund These are the only types that were reported as breaking changes. Are there any more changes I'm missing? I was expecting there are more public types to change.

Reply:
I would expect the WpfHost/GtkHost to be there 🤔 . Also these types should probably not be public - you can make them internal if possible

Reply:
@MartinZikmund WpfHost/GtkHost already have good casing.